modules/internal/OwnedObject.chpl

```diff
@@ -225,8 +207,9 @@ module OwnedObject {
        of the argument. The result has the same type as the argument.
        If the argument is non-nilable, it must be recognized by the compiler
        as an expiring value. */
+    @deprecated(notes="owned.create is deprecated - please use :proc:`owned.adopt` instead")
     inline proc type create(pragma "nil from arg" in take: owned) {
-      return take;
+      return owned.adopt(take);
```

Comment:
How are these two equivalent?  Is the new line unnecessary but sufficient?  (I guess, essentially a no-op?)

Reply:
These are the same because the `owned.adopt` implementation for owned is just
```chapel
proc adopt(take: owned) {
return take;
}
```

I figured it would be more clear replacing it with a call to adopt, but it amounts the same